<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Casos de COVID 19</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="dNILabel">
        <property name="text">
         <string>DNI</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_dni">
        <property name="inputMask">
         <string>99.999.999</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nombreLabel">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_nombre"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="apellidoLabel">
        <property name="text">
         <string>Apellido</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_apellido"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="edadLabel">
        <property name="text">
         <string>Edad</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_edad">
        <property name="inputMask">
         <string>009</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="sexoLabel">
        <property name="text">
         <string>Sexo</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cb_sexo">
        <item>
         <property name="text">
          <string>Hombre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mujer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Otros</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="provinciaLabel">
        <property name="text">
         <string>Provincia</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cb_provincia"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="fechaRegistroLabel">
        <property name="text">
         <string>Fecha Registro</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDateEdit" name="le_fecha_registro"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Persona</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_clear">
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_register">
        <property name="text">
         <string>Registrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clear_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_register</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>register()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clear_all()</slot>
  <slot>register()</slot>
 </slots>
</ui>
